app/helpers/message_helper.rb

```diff
@@ -0,0 +1,8 @@
+module MessageHelper
+  def smart_format(string, **options)
+    link_filter = AutoHtml::Link.new
+    with_links = link_filter.call(string)
+
+    simple_format(with_links, options)
```

Comment:
auto_html has it's own `AutoHtml::SimpleFormat` pipeline. Let's use that instead via a `AutoHtml::Pipeline`.